src/audit/unsecure_commands_allowed.rs

```diff
@@ -0,0 +1,117 @@
+use crate::audit::WorkflowAudit;
+use crate::finding::{Confidence, Finding, FindingBuilder, Severity, SymbolicLocation};
+use crate::models::{Steps, Workflow};
+use crate::state::AuditState;
+use github_actions_models::common::{Env, EnvValue};
+use github_actions_models::workflow::job::StepBody;
+use github_actions_models::workflow::Job;
+use std::ops::Deref;
+
+static ID: &str = "unsecure-commands-allowed";
```

Comment:
Just to make it shorter/fix GHA's own grammatical choice:

```suggestion
static ID: &str = "insecure-commands";
```

Reply:
Done [here](https://github.com/woodruffw/zizmor/pull/176/commits/f35682c69214f1f8e067af61d80acd434308f288). Also renamed some additional stuff to reflect that!